app/src/main/res/values/strings.xml

```diff
@@ -13,5 +13,6 @@
     <string name="category_technology">Технологии</string>
```

Comment:
Why do we mix English and Russian within the same resource file? Let's choose one and update others.